docs/resume.rst

```diff
@@ -176,12 +176,12 @@ operations consistently requires the following situation:
      snapshot.
 
      Even when the snapshot has been exported with a replication protocol
-     command, Postgres still requires the session to be maintained opened
+     command, Postgres still requires the session to be maintained as opened
```

Comment:
I think this change is wrong.

Reply:
Actually, it should be perhaps maintained as open here.". But it's OK, I reverted.